ansible/roles/test/tasks/qos_sai.yml

```diff
@@ -22,12 +22,36 @@
       minigraph_facts: host={{inventory_hostname}}
       become: no
 
+    - name: Get ports info.
+      include_tasks: roles/test/tasks/qos_get_ports.yml
+
+    - name: Check if lossless buffer profile is derived
+      fail: msg="Lossless Buffer profile could not be retreived"
+      when: lossless_buffer_profile is not defined or minigraph_hwsku is not defined
+
+    - set_fact:
+        defined_asic_list: ['td2', 'th', 'th2', 'spc1', 'spc2', 'spc3']
+        spc: "{{ lossless_buffer_profile }}"
```

Comment:
`spc` is a bit misleading. I've thought it stands for `Spectrum`. Is it possible to change to another name like `speed_cablelen`?

Reply:
done